app/controllers/api/irs_attempts_api_controller.rb

```diff
@@ -18,7 +18,12 @@ class IrsAttemptsApiController < ApplicationController
     respond_to :json
 
     def create
-      render json: { sets: security_event_tokens }
+      if timestamp
+        render json: { sets: security_event_tokens }
+      else
+        render json: { status: 422, description: 'Invalid timestamp parameter' },
```

Comment:
We *mostly* use the status names? WDYT about:

```suggestion
        render json: { status: :unprocessable_entity, description: 'Invalid timestamp parameter' },
```

And also... not to nitpick too much but a bad parameter sounds like a plain `:bad_request` (400) to me?

```suggestion
        render json: { status: :bad_request, description: 'Invalid timestamp parameter' },
```

Reply:
I initially had 400, but the spec suggests 422

Reply:
ok! maybe we should link to the spec doc in a comment?

Reply:
also wait I thing I misunderstood! is `status` a key inside the JSON payload in addition to the HTTP status?

Reply:
That's what the previous implementation had, I don't think it's strictly required though.